### ISSUE TITLE
semantic-ui hardcoded style for footer

### ISSUE THREAD

**Issue by kadirerdogan** [perak/kitchen-site#144]
@perak There is a hardcoded style for the semantic-ui footer in the meteor-kitchen binary "ui inverted center aligned vertical footer segment". 

Why do you need this to be hardcoded? I can't change footer style if the id is footer. 

Btw i think in a perfect world there shoud be no hardcoded style in the binary.

**Comment by perak** [comment]
I agree. If you don't spe ify any class for foter, cli adds default classes but that's bad practice (btw, if you specify anything, e.g. "foo" cli will not touch your classes). I'll remove that.

**Comment by kadirerdogan** [comment]
I didnt specify the class in the json file but the template has the class defined. 

I am trying to avoid all template specific declarations in the json file so that i can change the template with minimal changes.

**Comment by perak** [comment]
@kadirerdogan what do you suggest? 

1. To add default class into template, so if user not specify any class in json then class from template will be used

2. To leave as is - if no class defined in json, CLI defines class automatically.

3. To add application level swith: `"do_not_touch_css_classes": true` (default false)

... ?

There is also similar case with navbar and menus - if no class defined, default class is added.

Got idea how to avoid this? 

Goal is: if user doesn't specify any class in json - kitchen should produce (less or more) visually aesthetic and usable code. So, we definitely need some defaults here (in template or in CLI).
